client/components/common/BuildList.tsx

```diff
@@ -78,11 +94,16 @@ const BuildList: React.FC<Props> = ({
     setDeleteModalVisible(false);
   }, []);
 
-  const [search, setSearch] = React.useState('');
+  const { data: classData } = useQuery<classes>(classesQuery);
+  const { data: tagsData } = useQuery<customSetTags>(customSetTagsQuery);
+
+  const [search, setSearch] = React.useState(defaultSearch);
+
   const [dofusClassId, setDofusClassId] = React.useState<string | undefined>(
-    undefined,
+    defaultClassId,
```

Comment:
I know this is probably going to be annoying, but can we split this out for the state to live in the query parameter so there's a single source of truth? This seems pretty robust but in case there's a way for the query param to change by itself without the state changing, that could lead to bugs

Reply:
I am not seeing this one all that clearly, but I'll see what I can come up with to try to cover this condition.

The only problem I see with this is that the BuildList instance on the drawer (for the desktop version) doesn't generate query strings for what I explained in the PR's description, meaning it'd still need the state existing. 

I really don't see the query param changing without any 100% obvious user input and breaking, to be honest.